modules/administration/pages/ssl-certs-imported.adoc

```diff
@@ -81,10 +81,12 @@ To replace the certificates, you can replace the installed CA certificate with t
 
 .Procedure: Replacing Existing Certificates
 
-. On the {productname} Server, at the command prompt, call the command `mgr-ssl-cert-setup` and provide the certificates as parameters:
+. On the {productname} container host, at the command prompt, call the following command to run [command]``mgr-ssl-cert-setup`` inside the container and provide the certificates as parameters:
 +
 ----
-mgrctl exec -- mgr-ssl-cert-setup --root-ca-file=<Path_to_Root_CA_Certificate> --server-cert-file=<Server_Cert_File> --server-key-file=<Server_Key_File>
+mgrctl exec -ti -- mgr-ssl-cert-setup \
+  --root-ca-file=<Path_to_Root_CA_Certificate> \
+  --server-cert-file=<Server_Cert_File> --server-key-file=<Server_Key_File>
```

Comment:
You probably need to tell the user to copy the Root CA certificate and server key and cert in the container before calling that command: the paths will be inside the container, not on the host.
Also note that those paths are temporary ones: once the command has been called (and succeeded) they can be removed from the container.

Reply:
ok, committed from external.